tlsconfig/config_test.go

```diff
@@ -199,10 +199,10 @@ func TestConfigServerTLSClientCASet(t *testing.T) {
 // Exclusive root pools determines whether the CA pool will be a union of the system
 // certificate pool and custom certs, or an exclusive or of the custom certs and system pool
 func TestConfigServerExclusiveRootPools(t *testing.T) {
-	if runtime.GOOS == "windows" {
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
 		// FIXME TestConfigServerExclusiveRootPools is failing on windows:
 		// config_test.go:244: Unable to verify certificate 1: x509: certificate signed by unknown authority
-		t.Skip("FIXME: failing on Windows")
+		t.Skip("FIXME: failing on Windows and darwin")
```

Comment:
We should probably look at these tests in a follow-up. I recall I disabled them as they were broken, but I'm not sure if they are _expected_ to be failing (and if not; we should look "why")

Reply:
Yup; here is what I wrote in the commit message:

> TestConfigServerExclusiveRootPools and TestConfigClientExclusiveRootPools
are failing on darwin with the same error message as on Windows:
> 
> > Unable to verify certificate 1: x509: certificate signed by unknown authority
> 
> The `(*Certificate).Verify()` method from `crypto/x509` special-case
windows, darwin and ios GOOS to use a OS-specific verification process.
This process seems to consider root CAs as invalid for some unknown
reasons.
> 
> This should be further investigated.

Reply:
Yeah, I should've created a tracking issue when I added that skip; perhaps you can create one? (we could add a link to it in the source code / as a comment, in case more details arrive on GitHub)

Reply:
Done.